mycroft/skills/skill_loader.py

```diff
@@ -99,7 +99,8 @@ def _get_last_modified_time(path):
                 all_files.append(os.path.join(root_dir, f))
 
     # check files of interest in the skill root directory
-    return max(os.path.getmtime(f) for f in all_files)
+    if all_files:
+        return max(os.path.getmtime(f) for f in all_files)
```

Comment:
Should there be an 
```python
else
    return 0
```

I'd think a None would result in error when comparing with last_loaded

Reply:
Good point - changed

... and this is why I'm not doing the core release right now!

Reply:
Also it being 21:42 Friday evening I hope. Looks good now.